ui/page/components/items_scroll.go

```diff
@@ -68,6 +68,10 @@ func (s *Scroll[T]) FetchScrollData(isReverse bool, window app.WindowNavigator)
 	}
 	s.mu.Unlock()
 
+	// set isReset  to true here since all the callers of FetchScrollData()
+	// needs the data to be refreashed and data reset.
+	// TODO: set isreverse to default false all the callers of this
+	// FetchScrollData() is not initiating a reverse scroll action.
 	s.fetchScrollData(isReverse, window)
```

Comment:
This comment isn't clear. `set isReset  to true here`, what/where is `isReset`?
Since there's a follow up PR working on this file, let's keep changes here minimal and only absolutely necessary to reduce conflict and make this PR easier to follow.

Actually, I think undo all the changes to this file, including this comment and the `resetList` method name change.